privacy/consentvalidator.go

```diff
@@ -0,0 +1,14 @@
+package privacy
+
+type ConsentValidator interface {
+	ValidateConsent(consent string) bool
```

Comment:
Helps to create mocks in order to make the `readPolicy()` function's unit tests more robust

Reply:
This is test code, so please only keep it in test files so it's not compiled with the app for release.